dask/highlevelgraph.py

```diff
@@ -528,7 +534,9 @@ def layer_info_dict(self):
                 info[key] = html.escape(str(val))
         if self.collection_annotations is not None:
             for key, val in self.collection_annotations.items():
-                info[key] = html.escape(str(val))
+                # hiding chunks from the HTML table
```

Comment:
```suggestion
                # Hide verbose chunk details from the HTML table
```

Reply:
We want to explain the reason *why*, not just what the code is doing. (Reason being, the chunks information is often too verbose to be easily displayed in the HTML table)

Reply:
Yes, absolutely! Your suggestion is more appropriate and might be useful in the future if we decide to hide more properties.